### PR TITLE
Enabled Xlint scala option

### PR DESCRIPTION
**Pull request by magnolia-k**
and in addition, the cause of the newly outputted warning message was
removed.

### CODE REVIEW

File: src/test/scala/spray/json/StandardFormatsSpec.scala [hunk 1]
```diff
@@ -69,7 +69,7 @@ class StandardFormatsSpec extends Specification with DefaultJsonProtocol {
       (42, 4.2).toJson mustEqual json
     }
     "be able to convert a JsArray to a (Int, Double)]" in {
-      json.convertTo[(Int, Double)] mustEqual (42, 4.2)
+      json.convertTo[(Int, Double)] mustEqual Tuple2(42, 4.2)
```
Comment (ktoso) [inline src/test/scala/spray/json/StandardFormatsSpec.scala]: Simpler: `((42, 4.2))`

Comment (ktoso) [inline src/test/scala/spray/json/StandardFormatsSpec.scala]: Same everywhere else

Comment (magnolia-k) [inline src/test/scala/spray/json/StandardFormatsSpec.scala]: Thanks for review.

I fixed this code.

**Comment by ktoso** [comment]
Looks good

**Pull request merged**